sketch_map_tool/map_generation/qr_code.py

```diff
@@ -0,0 +1,82 @@
+import json
+from datetime import datetime, timezone
+from io import BytesIO
+from typing import Dict, List
+
+import qrcode
+import qrcode.image.svg
+from reportlab.graphics.shapes import Drawing
+from svglib.svglib import svg2rlg
+
+from sketch_map_tool import __version__
+
+
+def qr_code(
+    uuid: str,
+    bbox: List[float],
```

Comment:
Could be of type `Bbox` (from `helper_modules.bbox_utils`)

Reply:
Use new Bbox model class.